src/features/import/components/ImportDialog/Configure/Configuration/OrgConfig.tsx

```diff
@@ -16,9 +16,10 @@ interface OrgConfigProps {
 const OrgConfig: FC<OrgConfigProps> = ({ uiDataColumn }) => {
   const { orgId } = useNumericRouteParams();
   const messages = useMessages(messageIds);
-  const subOrgs = useSubOrganizations(orgId);
+  const subOrgs = useSubOrganizations(orgId).data || [];
+  const activeOrgs = subOrgs.filter((subOrg) => subOrg.is_active);
 
-  if (!subOrgs.data) {
+  if (!subOrgs) {
```

Comment:
For this to have the same effect as before, it needs to be changed to `if (!subOrgs.length)`. Before, `subOrgs` could be `null`, which is falsy. With your changes, it will instead be an empty array, and an empty array is truthy, so you need to check the length instead.

Reply:
Thank you!! 😸